phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -91,13 +95,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.common.base.Throwables;
 import com.google.common.collect.Maps;
 
 public class IndexRebuildRegionScanner extends GlobalIndexRegionScanner {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(IndexRebuildRegionScanner.class);
 
+    public static final String PHOENIX_LOG_BEYOND_MAX_LOOKBACK_ERROR_ROWS_CONF_KEY =
+        "phoenix.log.beyond.max.lookback.age.error.rows";
```

Comment:
Can we please have this config be named consistently like the previous one? 
"phoenix.mr.index.IndexDisableLoggingType=BEFORE" .. It would make it easier to find and put those together whenever they need non-default values.